### ISSUE TITLE
Browser does not proceed after MQTT setup

### ISSUE THREAD

**Issue by anantbhula** [marvinroger/homie#6]
Hi First of all big thank you for the homie. That is what i have been looking for. I was running ESPEasy for a while but seems like it is highly unreliable. My issue with Homie currently is I am having trouble saving config at MQTT setup point. Even I click Next the browser does not move.
I have tried https://community.openhab.org/t/framework-homie-for-esp8266/8674/37
and re-uploaded https://github.com/marvinroger/homie-esp8266/blob/develop/examples/IteadSonoff/IteadSonoff.ino
I can see in serial monitor everything is working until MQTT setup after entering all the information it does not move to the next stage. Am I missing something?
I have tried on Safari, Chrome, Firefox and Internet Explorer all same result. Please help

**Comment by marvinroger** [comment]
What version of `homie-esp8266` are you using? Which configurator do you use? (the URL?)

**Comment by anantbhula** [comment]
homie-esp8266 ---> v1.5.0 from https://github.com/marvinroger/homie-esp8266/releases/tag/v1.5.0
URL - http://marvinroger.github.io/homie-esp8266/

**Comment by marvinroger** [comment]
Can you post a screenshot of the interface at the moment it fails?

**Comment by anantbhula** [comment]
After you asked which version homie I downloaded develop version from githubhttps://github.com/marvinroger/homie-esp8266 uploaded and it passed MQTT section. Configuration saved but after reboot it is not connecting to my AP to go live.

**Comment by anantbhula** [comment]
![screen shot 2016-10-09 at 8 22 54 am](https://cloud.githubusercontent.com/assets/14354228/19220362/a68826a4-8df9-11e6-8c44-2023a6783dc4.png)
![screen shot 2016-10-09 at 8 23 05 am](https://cloud.githubusercontent.com/assets/14354228/19220363/a688bc4a-8df9-11e6-8844-d01bb8e31286.png)
Screenshot of Serial monitor and MQTT.fx after reboot nothing is happening and I have my AP connected device list is open as well but nothing popping up ATM.
Thanks for you help,

**Comment by marvinroger** [comment]
Don't mix the v2 and the configurator of the v1. Try again with the v1.5.0 and send a screenshot of the filled MQTT so I can reproduce.

**Comment by anantbhula** [comment]
Will do and let you know

> Don't mix the v2 and the configurator of the v1. Try again with the v1.5.0 and send a screenshot of the filled MQTT so I can reproduce.

What you mean by don't mix v2 and configurator of v1? V2 does not require configurator? How do I upload configuration in v2?

**Comment by anantbhula** [comment]
I got it working... The only thing I have done different was to unhook from serial connection and left it as standalone and I started to see messages in MQTT.fx and then it just a matter of time till i figured out from your great explanation about convention.
![screen shot 2016-10-09 at 10 09 12 am](https://cloud.githubusercontent.com/assets/14354228/19221002/7420b9e2-8e08-11e6-93a6-45198b75de25.png)

Appreciate your help and hats off to you for great project keep it up. Looks way stable than ESPEasy. If i run into problem will let you know.
